backend/flindt/feedback/models.py

```diff
@@ -166,7 +166,7 @@ def save(self, *args, **kwargs):
 
         def send_feedback_received_message():
             message = _(
-                'You just received feedback. Read and rate it! https://{}/#/received-feedback/{}'.
+                'You just received feedback. Read and rate it! https://{}/received-feedback/{}'.
```

Comment:
Would normally put this in a separate ticket.

Reply:
Yeah, I thought it was (and merged already) but I must've confused two branches before. #oops